shopfloor_example/services/partner_service.py

```diff
@@ -65,17 +65,17 @@ class ShopfloorCheckoutValidator(Component):
     _name = "shopfloor.partner_example.validator"
     _usage = "partner_example.validator"
 
-    def _validator_scan(self):
+    def scan(self):
```

Comment:
hmm I  think I forgot to rename them after moving to their own component :sweat_smile: